dubbo-remoting/dubbo-remoting-zookeeper/src/main/java/org/apache/dubbo/remoting/zookeeper/curator/CuratorZookeeperClient.java

```diff
@@ -74,6 +74,10 @@ public void stateChanged(CuratorFramework client, ConnectionState state) {
         }
     }
 
+    public CuratorZookeeperClient(URL url) {

```

Comment:
I this `this(url,new HashSet<>());` would better. What do you say?

Reply:
Order wise less parameters constructor better to come before then more once. Should we place single argument before two parameters constructor for better visibility.

Reply:
I have remove this method for refactoring the code.  Thanks for you reply.